fix: CI build fails — @barodoc/theme-docs missing from build:packages

## Problem

CI docs build fails with:

```
Cannot find module '@barodoc/theme-docs/theme' imported from 'docs/astro.config.mjs'
```

## Cause

PR #85 added `tsup.config.ts` to `@barodoc/theme-docs`, making it require a build step to produce `dist/theme.js`. However, `@barodoc/theme-docs` was not added to the `build:packages` script in root `package.json`.

## Fix

Add `--filter @barodoc/theme-docs` to the `build:packages` script.

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
     "dev": "pnpm --filter docs dev",
     "dev:docs": "pnpm --filter docs dev",
     "build": "pnpm --filter docs build",
-    "build:packages": "pnpm --filter @barodoc/core --filter barodoc --filter create-barodoc --filter @barodoc/plugin-sitemap --filter @barodoc/plugin-search --filter @barodoc/plugin-analytics --filter @barodoc/plugin-openapi --filter @barodoc/plugin-rss --filter @barodoc/plugin-pwa --filter @barodoc/plugin-llms-txt --filter @barodoc/plugin-og-image --filter @barodoc/plugin-docsearch build",
+    "build:packages": "pnpm --filter @barodoc/core --filter @barodoc/theme-docs --filter barodoc --filter create-barodoc --filter @barodoc/plugin-sitemap --filter @barodoc/plugin-search --filter @barodoc/plugin-analytics --filter @barodoc/plugin-openapi --filter @barodoc/plugin-rss --filter @barodoc/plugin-pwa --filter @barodoc/plugin-llms-txt --filter @barodoc/plugin-og-image --filter @barodoc/plugin-docsearch build",
     "test": "vitest run",
     "test:watch": "vitest",
     "typecheck": "pnpm -r typecheck",

